packages/eyes/src/generate-snapshots/build-base-files.ts

```diff
@@ -0,0 +1,47 @@
+import Registry, {importMeta} from '@ui-autotools/registry';
+import {consoleLog} from '@ui-autotools/utils';
+import fs from 'fs';
+import path from 'path';
+import {generateFilename, generateData} from './filename-utils';
+
+export const buildBaseFiles = (projectPath: string) => {
+  consoleLog('Building base files...');
+  consoleLog('Importing Metafiles...');
+  importMeta();
+
+  const autotoolsFolder = path.join(projectPath, '.autotools');
+
+  if (!fs.existsSync(autotoolsFolder)) {
+    fs.mkdirSync(autotoolsFolder);
+  }
+
+  const stylePathPrefix = '../';
+  const compPathPrefix = '../';
+
+  Registry.metadata.components.forEach((componentMetadata) => {
+    const numberOfSims = componentMetadata.simulations.length;
+    const styles = componentMetadata.styles;
+    const compPath = compPathPrefix + componentMetadata.path;
+    const compName = componentMetadata.exportName;
+
+    if (compName) {
+      for (let i = 0; i < numberOfSims; i++) {
+        const simulationName = componentMetadata.simulations[i].title;
+        if (styles.size) {
+          let styleIndex = 1;
+          styles.forEach((style) => {
```

Comment:
Instead of defining styleIndex above and incrementing in manually you could do:
```
styles.forEach((style, styleIndex) => {
    generateFilename(..., styleIndex + 1);
})
```